instrumentation/jaxrs2/src/test/java/brave/jaxrs2/features/sampling/Traced.java

```diff
@@ -0,0 +1,37 @@
+package brave.jaxrs2.features.sampling;
+
+import brave.http.HttpAdapter;
+import brave.http.HttpSampler;
+import brave.jaxrs2.ContainerAdapter;
+import brave.sampler.DeclarativeSampler;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.reflect.Method;
+import javax.annotation.Nullable;
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Traced {
```

Comment:
Thanks, @adriancole, the annotation is useful baseline, could be interpreted by frameworks / apps differently